archives.php

```diff
@@ -224,9 +224,9 @@ protected function getArchives($filters, $operator, $order)
                 }
             }
             if ($new_approach) {
-                $collection = $page->children();
+                $collection = $page->collection();
```

Comment:
collection() alone is not enough. pagination in the collection definition will cause only the first page of results to be taken into account for the archives.

Reply:
Thanks for the review, I'll revisit this PR when I have some free time (probably next week) and check if disabling pagination works!

Reply:
Disabled pagination in latest version, seems to be working as intended 👌

Reply:
Bumping this -- don't hesitate to HMU if I can do anything.